tests/test_non_dates.py

```
@@ -0,0 +1,94 @@
+class TestNonDateCases:
+    """Test cases where non-date indexes are used"""
+
+    @pytest.mark.core
+    def test_integer_index(self):
+        """Test standard gradual trend but with no date index."""
+        df = pt.load_data('series_synthetic')
+        results = pt.detect_trends(
+            df,
+            value_col='gradual',
+            plot=False,
+            method_params=dict(is_abrupt_padded=False)
```

Ruff (0.15.20)

[warning] Unnecessary `dict()` call (rewrite as a literal)

Rewrite as a literal

(C408)